### PR TITLE
feat(SlackIcon): Added slack icon

### PR DESCRIPTION
**Pull request by MattSchiller**
### Overview

Adding Slack icon to icons list

Using it for Nexus, figured why not

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.2.2-alpha.ce0298.0<br/>@codecademy/gamut-kit@0.6.124-alpha.ce0298.0<br/>@codecademy/gamut-labs@31.11.13-alpha.ce0298.0<br/>@codecademy/gamut@47.2.2-alpha.ce0298.0<br/>@codecademy/monaco@0.6.43-alpha.ce0298.0<br/>@codecademy/styleguide@57.1.2-alpha.ce0298.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://627c0362bf60771286db4ac7--gamut-preview.netlify.app>https://627c0362bf60771286db4ac7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/627c0362bf60771286db4ac7>Deploy Logs</a></p>

File: packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx [hunk 1]
```diff
@@ -94,7 +94,7 @@ General user interface icons with affordances
   ))}
 </IconGallery>
 
-### Learning Enviornment
+### Learning Environment
```
Comment (aresnik11) [inline packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:97]: lol

Comment (MattSchiller) [reply packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:97]: Yeah, this was bothering me when I saw it in the storybook, so I snuck it in.

**Review by callafeucht — APPROVED** [review]
I don't know that I'm like.. authorized to approve these? But I approve. Big thumbs up. 👍

**Review by dreamwasp — APPROVED** [review]

**Pull request merged**